app/index.coffee

```diff
@@ -105,6 +107,11 @@ Api.proxy.el().one 'load', ->
       app.topBar.loginForm.signInButton.click()
       app.topBar.loginDialog.reattach()
 
+    $(window).bind('beforeunload', (e) ->
+        AnalyticsLogger.logEvent User.current?.zooniverse_id,Subject.current?.zooniverseId,'leave',''
+        event.preventDefault()
```

Comment:
i was trying to ensure that no popup appears. i thought it was this line that did the trick, but maybe not. will edit or remove. thanks.